### ISSUE TITLE
Openfoam installation - "docker-machine.exe" is not recognized

### ISSUE THREAD

**Issue by cyshimz** [ladybug-tools/butterfly#347]
Hi, I'm trying to install Openfoam on my pc. I've resolved all the errors I've encountered with the Docker Quickstart and OF-Env_Create, but I got stuck at the actual OpenFOAM_Start.

![image](https://user-images.githubusercontent.com/39609051/40523420-0312508e-6008-11e8-9bb9-8d38a98cb7b1.png)

Can anybody help? Much thanks!

**Comment by DarylGraves** [comment]
Are you able to share how fixed this issue?

**Comment by cyshimz** [comment]
Hi @DarylGraves, I can't remember how I fixed this one particular issue as the entire installation process was very buggy and it took a fair bit of troubleshooting. Here's what I remember eventually working for me (Windows 10, Rhino 6):

1. I redownloaded and installed VirtualBox 5.2.0 separately instead of using the files in the installation package
2. Running all the relevant programs, including Rhino as admin (right click, Run as administrator)
3. Running the related programs using the same user profile that the butterfly folder is located in - under Users/*profile name*/butterfly <-- this folder is where all the butterfly simulations will  be located in
4. Using Openfoam 1612+ instead of any other version
5. Using Butterfly 0.04 instead of any other version

The above error showed up when I used Openfoam 1712, btw. Try 1612+! 

Hope this helps!
